src/accordion.tsx

```diff
@@ -0,0 +1,346 @@
+    index?: number;
+    defaultIndex?: number;
+};
+
+type MultipleOpenProps = {
+    singleOpen?: false;
+    index?: number | Array<number>;
+    defaultIndex?: number | Array<number>;
+};
+
+type AccordionProps = AccordionBaseProps & ExclusifyUnion<SingleOpenProps | MultipleOpenProps>;
+
+export const Accordion: React.FC<AccordionProps> = ({
+    children,
+    dataAttributes,
+    noLastDivider,
+    index,
+    defaultIndex,
+    onChange,
+    singleOpen,
+}) => {
+    const [indexList, toogle] = useAccordionState({
+        value: index,
+        defaultValue: defaultIndex,
+        onChange,
+        singleOpen,
+    });
+    const lastIndex = React.Children.count(children) - 1;
+    const showLastDivider = !noLastDivider;
+
+    return (
+        <AccordionContext.Provider value={{index: indexList, toogle}}>
+            <div {...getPrefixedDataAttributes({...dataAttributes, accordion: true}, 'Accordion')}>
+                {React.Children.toArray(children)
+                    .filter(Boolean)
+                    .map((child, index) => (
+                        <React.Fragment key={index}>
+                            {child}
+                            {(index < lastIndex || showLastDivider) && (
+                                <Box paddingX={16}>
+                                    <Divider />
+                                </Box>
+                            )}
+                        </React.Fragment>
+                    ))}
+            </div>
+        </AccordionContext.Provider>
+    );
+};
+
+interface BoxedAccordionItemProps extends AccordionItemContentProps {
+    isInverse?: boolean;
+}
+
+export const BoxedAccordionItem = React.forwardRef<HTMLDivElement, BoxedAccordionItemProps>(
+    ({dataAttributes, isInverse, ...props}, ref) => (
+        <Boxed
+            isInverse={isInverse}
+            ref={ref}
+            dataAttributes={{'component-name': 'BoxedAccordionItem', ...dataAttributes}}
+        >
+            <AccordionItemContent {...props} />
+        </Boxed>
+    )
+);
+
+type BoxedAccordionProps = Omit<AccordionProps, 'noLastDivider'>;
+
+export const BoxedAccordion: React.FC<BoxedAccordionProps> = ({
+    children,
+    dataAttributes,
+    index,
+    defaultIndex,
+    onChange,
+    singleOpen,
+}) => {
+    const [indexList, toogle] = useAccordionState({
+        value: index,
+        defaultValue: defaultIndex,
+        onChange,
+        singleOpen,
+    });
+
+    return (
+        <AccordionContext.Provider value={{index: indexList, toogle}}>
+            <Stack
+                space={16}
+                dataAttributes={{'component-name': 'BoxedAccordion', accordion: true, ...dataAttributes}}
+            >
+                {children}
+            </Stack>
+        </AccordionContext.Provider>
+    );
+};
+
+type GroupedAccordionProps = Omit<AccordionProps, 'noLastDivider'> & {isInverse?: boolean};
+
+export const GroupedAccordion: React.FC<GroupedAccordionProps> = ({
```

Comment:
Should we have `GroupedAccodion`? According to the specs:

![image](https://github.com/Telefonica/mistica-web/assets/25785151/880055d2-edb0-497b-855a-1ae572bb8801)

But after discussing with @yceballost, we are not sure this should be provided by Mistica (it's just wrapping the Accordion inside a Boxed component and setting `noLastDivider=true`). If we decide to keep this component, we should create the equivalent one for `RowList`.

Reply:
imo, we don't need <GroupedAccordion />, just put accordion inside a `<Boxed />` as the same way with the lists

Reply:
Ok, it makes sense. Removing GroupedAccordion